source/includes/fundamentals/code-snippets/wildfly-module-4.0.0.xml

```diff
@@ -0,0 +1,10 @@
+<module xmlns="urn:jboss:module:1.3" name="org.mongodb">
+   <resources>
+      <resource-root path="mongodb-driver-sync-4.0.0.jar"/>
```

Comment:
No need to change at the moment; 4.1.0 is the latest driver.
Also for the filename, I meant the postfix name would be "-with-version" so we could find filenames that included "-with-version". No need to change this or implement any naming scheme in this PR.

Reply:
Gotcha, updated.